packages/contracts/contracts/test/plugin-mock/Clonable/PluginClonableMock.sol

```diff
@@ -1,15 +0,0 @@
-// SPDX-License-Identifier: MIT
-
-pragma solidity 0.8.10;
-
-import {PluginCloneable} from "../../../core/plugin/PluginCloneable.sol";
-import {IDAO} from "../../../core/IDAO.sol";
-
-contract PluginCloneableMock is PluginCloneable {
```

Comment:
is contribution history is lost because of the typo in the file name ?

Reply:
The file will appear as a new file and the old one will appear as deleted. The history is interrupted. Looking at the new files, you wouldn't see the previous authors.

We would need to rename/move in a different commit and change the contents in a subsequent commit.
However, with our current practice, if we "squash and merge" in the end, everything is squashed into one commit again, and it would be lost anyway. 

I think we should change this "squash and merge" practice @mathewmeconry or do renaming/removing commits in separate PRs @mathewmeconry.

Reply:
"squash and merge" was the standard. if we wanna change I am happy with it. I am more a fan of normal merge commits

Reply:
@Rekard0 will start the discussion. For now, we can proceed.